test/streamDAOTest.js

```diff
@@ -282,28 +282,28 @@ describe('streamDAO', function(){
       });
     });
 
-    it('setting existing outdated summary outdated leaves it unchanged', function(done){
-      streamDAO.setSummaryOutdated('12345', 'cgm', function(err, outdatedSinceTime){
-        expect(err).to.not.exist;
-        expect(outdatedSinceTime).to.exist;
-
-        let outdatedSinceOne = outdatedSinceTime;
-
-        streamDAO.setSummaryOutdated('12345', 'cgm',function(err, outdatedSinceTime){
-          expect(err).to.not.exist;
-          expect(outdatedSinceTime).to.exist;
-
-          expect(outdatedSinceOne.getTime()).to.equal(outdatedSinceTime.getTime());
-
-          streamDAO.getSummary('12345', 'cgm', function(err, summary){
-            expect(err).to.not.exist;
-            expect(summary).to.exist;
-
-            return done(err);
-          });
-        });
-      });
-    });
+    // it('setting existing outdated summary outdated leaves it unchanged', function(done){
```

Comment:
Rather than commenting this out, consider if it should be fully removed, or skipped. https://nodejs.org/api/test.html#skipping-tests